inscriptions/ISic000547.xml

```diff
@@ -199,88 +204,23 @@
-                    <lb n="8"/>
-                    <expan>
-                        <abbr>her</abbr>
-                        <ex>es</ex>
-                    </expan>
-                    <expan>
-                        <abbr>b</abbr>
-                        <ex>ene</ex>
-                    </expan>
-                    <expan>
-                        <abbr>m</abbr>
-                        <ex>erenti</ex>
-                    </expan>
-                    <expan>
-                        <abbr>f</abbr>
-                        <ex>ecit</ex>
-                    </expan>
-                    .
+                    <lb n="1"/><expan><abbr>d</abbr><ex>is</ex></expan> <expan><abbr>m</abbr><ex>anibus</ex></expan>
+                    <lb n="2"/><expan><abbr><supplied reason="lost">A</supplied>ur</abbr><ex>elio</ex> <g ref="#interpunct">·</g> </expan>Rustico<expan> <g ref="#interpunct">·</g> <abbr>mil</abbr><ex>itavit</ex></expan>
+                    <lb n="3"/><expan><abbr>ann</abbr><ex>os</ex></expan> <g ref="#interpunct">·</g> VIII <expan><abbr>tr</abbr><ex>iere</ex></expan> <g type="centuria">centuria</g><!--glyph looks like a backwards 'Y'?--> Zeno
+                    <lb n="4" break="no"/>nis <g ref="#interpunct">·</g> <expan><abbr>Mar</abbr><ex>cus</ex> </expan> <g ref="#interpunct">·</g> Germanici
+                    <lb n="5" break="no"/>anus <g ref="#interpunct">·</g> <expan><abbr>mil</abbr><ex>itavit</ex> <g ref="#interpunct">·</g> </expan><expan><abbr>ann</abbr><ex>os</ex></expan> <g ref="#interpunct">·</g> X
+                    <lb n="6"/><expan><abbr>Valer</abbr><ex>ius</ex></expan> <g ref="#interpunct">·</g> Victorinus
+                    <lb n="7"/><expan><abbr>her</abbr><ex>es</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>b</abbr><ex>ene</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>m</abbr><ex>erenti</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ecit</ex></expan>
                 </ab>
-                <!--This EDR text 138303 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
             </div>
             <div type="apparatus">
                 <p><!--external apparatus criticus (if applicable)--></p>
             </div>
-            <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+            <div type="translation" xml:lang="en" resp="#NCash">
+                <!--I was unsure of how to accurately translate 'triere centuria'; also, seeing as 'heres' and 'fecit' are singular, I thought that Germanicianus could not be a dedicant alongside Valerius, but I do not understand why his name is in the nominative-->
+                <p>To the spirits of the Underworld. Valerius Victorinus, the heir, made this for the well-deserving Aurelius Rusticus, who served for eight years in the century aboard the trireme of Zenon, (and for) Marucs Germanicianus, who served for ten years.</p>
```

Comment:
I think you have this right, other than that it is 'the trireme Zenon' (i.e. it's the name of the trireme), and typo on Marcus. The case shift is awkward, but happens with some frequency. The alatnerative would be to make the last line plural, which one could do.